puppet/modules/tapicero/manifests/init.pp

```diff
@@ -133,4 +133,17 @@
                     Couchdb::Add_user[$::site_couchdb::couchdb_tapicero_user] ];
   }
 
+  rsyslog::snippet { '99-tapicero':
+    content => 'if $programname startswith \'tapicero\' then /var/log/leap/tapicero.log
+&~'
+  }
+
+  augeas {
+    'logrotate_tapicero':
+      context => '/files/etc/logrotate.d/tapicero/rule',
+      changes => [ 'set file /var/log/leap/tapicero*.log', 'set rotate 7',
+                   'set schedule daily', 'set compress compress',
+                   'set missingok missingok', 'set ifempty notifempty',
+                   'set copytruncate copytruncate' ]
+  }
 }
```

Comment:
in site_webapp/manifests/logging.pp

    rsyslog::snippet { '01-webapp':
      content => 'if $programname == "webapp" then /var/log/leap/webapp.log
    &~'
    }